### ISSUE TITLE
CorsikaQuery (pytables.read_where) does not work for 'log10(energy)=1x.5'

### ISSUE THREAD

**Issue by tomkooij** [HiSPARC/sapphire#114]
All ".5" energy [15.5, 16.5, 17.5] are not found by CorsikaQuery (even though they are in ```corsika_overview.h5```

Example:
```
>>> cq.simulations(energy=15.5)
array([], dtype=

```

This is because of pytables.read_where():

```
`>>> data.root.simulations.read_where('log10(energy)==16.5')
array([], dtype=
```

Integer energies do work as expected.

```
>>> tables.__version__
'3.2.2'
```

**Comment by 153957** [comment]
Ok, so should probably use the `float_filter` which is also used for `azimuth` or `zenith`.

**Comment by 153957** [comment]
Hmm, I do not have this problem, please try this first:
`cq.available_parameters('energy')`
Does the resulting set include `.5` values?

**Comment by tomkooij** [comment]
```
In [7]: cq.available_parameters('energy')
Out[7]: {12.0, 12.5, 13.0, 13.5, 14.0, 14.5, 15.0, 15.5, 16.0, 16.5, 17.0, 17.5, 18.0}
```
This is on ```stbc-i1``` so you can reproduce it if you ```source activate tom```

**Comment by 153957** [comment]
You are using the overview from /data/hisparc/corsika/ ?
(I do not have this problem locally, also pytables 3.2.2)

**Comment by tomkooij** [comment]
Yes: this is the script: https://gist.github.com/tomkooij/03a45b3e828373663993

The only way I can make it work is by quering for energy instead of log10(energy). I can make a PR if you want.

**Comment by 153957** [comment]
Fixed in 4eb8203
